### DISCUSSION TITLE
Dissemination

### DISCUSSION THREAD

**Discussion by clizbe** [TulipaEnergy/TulipaEnergyModel.jl#649] [Ideas]
Here's my plan for dissemination, because right now we only have the github to send people to:
1. Make an attractive & friendly overview page on Energy.nl in the Tools section that links to Github
2. When we want to, make a news article on the TNO website and/or LinkedIn, linking to the Energy.nl page

So what do we want on the Energy.nl page? It should be friendly but interesting and preferably not need updating. It's a landing page for newcomers to get them excited enough to check out the Github - or in the future, read the analyses.

@gnawin @g-moralesespana @datejada @abelsiqueira @suvayu @greg-neustroev

**Comment by clizbe** [comment]
We could also maybe use the strategy on[ this page](https://energy.nl/tools/investments-in-energy-efficiency/) and have a main page that links to two other pages:

1. For Energy Modellers (tech stuff)
2. For Stakeholders (impact stuff)

**Comment by gnawin** [comment]
From a user point of view, I'd like to see something both easy-to-read and eye-catching. So I'd put a nice picture with a few words as headlines (for landing) and use the pamphlet (or its modified version or some parts of the poster, you get what I mean) as actual content.

**Comment by suvayu** [comment]
Maybe a distilled version of the poster?

-- 
Suvayu

Open source is the future. It sets us free.

**Comment by clizbe** [comment]
Sounds good. Added to my To-Do list.
